gitfourchette/repowidget.py

```diff
@@ -777,6 +785,19 @@ def refreshPrefs(self, prefDiff: list[str]):
         # Reflect any change in titlebar prefs
         self.refreshWindowTitle()
 
+    def onAutoFetchTimerTimeout(self):
+        if not settings.prefs.autoFetch or not self.isVisible() or self.taskRunner.isBusy():
+            return
+
+        # Check if it's time to auto-fetch.
+        now = time.time()
+        interval = min(1, settings.prefs.autoFetchMinutes) * 60
```

Comment:
```suggestion
        interval = max(1, settings.prefs.autoFetchMinutes) * 60
```
I think this should enforce a 1-minute interval if the user has a <= 0 value in their prefs.